style.css

```diff
@@ -102,3 +91,308 @@ p {
   align-items: center;
   margin: 0% 0% 0% 7%;
 }
+
+.headline-tagline {
+  height: 519px;
+  background-image: url("./images/Group.png");
+  background-position-x: right;
+  background-position-y: bottom;
+  background-repeat: no-repeat;
+  align-items: flex-start;
+  bottom: -11.63%;
+  left: 65.74%;
+  right: -30.09%;
+  top: 29.3%;
+}
+
+#MyRecentWorks {
+  font-family: 'Crete Round', serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 40px;
+  line-height: 52px;
+  margin-top: 120px;
+  text-align: center;
+}
+
+#hr-line {
+  width: 327px;
+  height: 0;
+  border: 1px solid #212121;
+  align-items: center;
+  margin-left: 15px;
+}
+
+.main-div {
+  display: flex;
+  flex-direction: column;
+  align-items: flex-start;
+  background-color: white;
+  padding: 10px;
+  gap: 24px;
+}
+
+.top-div {
+  display: flex;
+  flex-direction: column;
+  gap: 24px;
+}
+
+#yoga-image {
+  width: 327px;
+  height: 237px;
+  margin-top: 56px;
+  margin-left: 15px;
+}
+
+.butm-div {
+  display: flex;
+  flex-direction: column;
+  align-items: flex-start;
+}
+
+h2 {
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 32px;
+  line-height: 44px;
+  margin: 5px 10px 10px 20px;
+}
+
+.butm-div h2 {
+  font-family: 'Crete Round', serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 32px;
+  line-height: 44px;
+}
+
+.butm-div p {
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 16px;
+  line-height: 24px;
+}
+
+.list1 {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  margin-left: 18px;
+  gap: 10px;
+}
+
+.list2 li {
+  list-style: none;
+  background: rgba(255, 255, 255, 0.24);
+  padding: 10px 12px;
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 15px;
+  line-height: 20px;
+}
+
+.list3 li {
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 22px;
+  line-height: 43px;
+  list-style: none;
+}
+
+.butm-div .list1 li {
+  list-style: none;
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 15px;
+  line-height: 20px;
+  border: 1px solid #8993a4;
+  background: #fff;
+  padding: 10px 12px;
+}
+
+.butm-div a {
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 17px;
+  line-height: 24px;
+  padding: 12px;
+  background: #ff6b00;
+  width: 124px;
+  height: 48px;
+  color: #fff;
+  letter-spacing: 0.03em;
+  text-decoration: none;
+  margin-top: 10px;
+  margin-left: 18px;
+  white-space: nowrap;
+}
+
+.project_list {
+  background-color: white;
+  display: grid;
+  grid-template-rows: repeat(6, auto);
+  border: 0;
+  gap: 24px;
+}
+
+.project2 {
+  display: flex;
+  flex-direction: column;
+  background-image: url(images/placeholder.png);
```

Comment:
You made a great attempt to fix the changes that the previous reviewer requested 👍 . However, the problem still persists. I did some digging however and I realized that the image you are trying to access is being spelled wrongly. It starts with a capital letter and hence this line should look something like this: 

```
background-image: url(images/Placeholder.png);
```